### PR TITLE
docs: add triage label maintainer documentation

### PR DESCRIPTION
**Pull request by cdrage**
docs: add triage label maintainer documentation

### What does this PR do?

Added documentation on how to triage issues.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -353,3 +353,17 @@ If you're unsure where to add code (renderer, UI, extensions, plugins) see the b
...
 
 See our [EXTENSIONS.md](/EXTENSIONS.md) document for more details.
+
+## Maintainer tasks
+
+List of maintainer tasks to help the project run smoothly.
+
+### Triaging
+
+New issues will have the [triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label assigned. As a maintainer, you'll have to curate these issues into the appropriate [area labels](https://github.com/containers/podman-desktop/labels).
```
Comment (benoitf) [inline CONTRIBUTING.md]:
```suggestion
New issues will have the [status/need-triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label assigned. As a maintainer, you'll have to curate these issues into the appropriate [area labels](https://github.com/containers/podman-desktop/labels).
```

**Review by benoitf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -353,3 +354,17 @@ If you're unsure where to add code (renderer, UI, extensions, plugins) see the b
...
+
+Ideally complete the task every morning:
+
+1. [View all the current triage labeled issues.](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage)
+2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
+3. Remove the `status/need-triage` label.
```
Comment (odockal) [inline CONTRIBUTING.md:370]: How about assignment of the issue to a concrete person? How about possible high severity issues? That is out of scope for triaging? Just curious...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -353,3 +354,17 @@ If you're unsure where to add code (renderer, UI, extensions, plugins) see the b
...
+List of maintainer tasks to help the project run smoothly.
+
+### Triaging
+
+New issues will have the [triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label assigned. As a maintainer, you'll have to curate these issues into the appropriate [area labels](https://github.com/containers/podman-desktop/labels).
+
+Ideally complete the task every morning:
+
+1. [View all the current triage labeled issues.](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage)
+2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
```
Comment (themr0c) [inline CONTRIBUTING.md]: Consider displaying only labels containing `area/`:

```suggestion
2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels?q=area%2F) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (themr0c) [reply CONTRIBUTING.md]: Consider also displaying only labels containing `area/`:

```suggestion
New issues will have the [status/need-triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label assigned. As a maintainer, you'll have to curate these issues into the appropriate [area labels](https://github.com/containers/podman-desktop/labels?q=area%2F).
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (themr0c) [reply CONTRIBUTING.md]: We have 39 labels! For someone triaging for the first time it might be overwhelming.

Can we consider simplifying this list?

For docs, when we design steps in a procedure, we try (sometimes hard) to stick to the "rule of seven", as it is the maximum a human brain is supposed to be able to process in one go. Getting closer to 7 could be beneficial here too.

We could:

- Keep the 7 label are used for more than 20 issues.
- Remove the 12 labels are unused or used for one unique issue.
- Try to regroup the remaining labels.
- Add a description to the labels, with the triage person in mind.

**Comment by benoitf** [comment]
@cdrage what is the state of this PR ? include @odockal and @themr0c feedback ?

File: CONTRIBUTING.md (hunk 2 above)
Comment (cdrage) [reply CONTRIBUTING.md:370]: Out of scope, it's kinda simple right now and we haven't created a process for triaging to another person yet.

**Comment by cdrage** [comment]
> @cdrage what is the state of this PR ? include @odockal and @themr0c feedback ?

Updated per feedback from @themr0c and @odockal

**Comment by cdrage** [comment]
Resolved the issues, since this has already been approved and it's only for a small document (CONTRIBUTING.md), I'm going to merge 👍
